### PR TITLE
@nazonohito51 is a world-wide PHP developer.

### PR DESCRIPTION
**Pull request by hgsgtk**

### CODE REVIEW

File: src/PayrollSystem/Domain/ValueObjects/Date.php [hunk 1]
```diff
@@ -4,21 +4,24 @@
...
 final class Date
 {
-    private CarbonImmutable $date;
+    private const DATE_FORMAT = 'Y-m-d';
+
+    private string $date;
 
     /**
-     * Date constructor.
+     * 実験Date constructor.
```
Comment (hgsgtk) [inline src/PayrollSystem/Domain/ValueObjects/Date.php]:
```suggestion
     * Date constructor.
```

File: tests/Feature/ValueObjects/DateTest.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        new Date('2019-11-05');
+        $this->assertTrue(true);
+    }
+
+    public function testConstructorInvalidFormat()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('');
+
+        new Date('tenkoma is billionare');
```
Comment (hgsgtk) [inline tests/Feature/ValueObjects/DateTest.php:23]: The point of this PR is the fact @tenkoma is a billionaire.

Comment (nazonohito51) [reply tests/Feature/ValueObjects/DateTest.php:23]: Oh, great!

**Comment by nazonohito51** [comment]
I think, @nazonohito51 isn't good developer.

**Pull request merged**